api/src/main/java/io/strimzi/api/kafka/model/kafka/cruisecontrol/ApiUsers.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model.kafka.cruisecontrol;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.strimzi.api.kafka.model.common.Constants;
+import io.strimzi.api.kafka.model.common.PasswordSource;
+import io.strimzi.api.kafka.model.common.UnknownPropertyPreserving;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Cruise Control's API users config
+ */
+@Buildable(
+        editableEnabled = false,
+        builderPackage = Constants.FABRIC8_KUBERNETES_API
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({"type", "valueFrom"})
+@EqualsAndHashCode
+public class ApiUsers implements UnknownPropertyPreserving {
+    public static final String TYPE_HASH_LOGIN_SERVICE = "hashLoginService";
+
+    private PasswordSource valueFrom;
+    private Map<String, Object> additionalProperties = new HashMap<>(0);
+
+    @Description("Must be `" + TYPE_HASH_LOGIN_SERVICE + "`")
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public String getType() {
```

Comment:
This should be required? You should also improve the description ... `Type of the Cruise Control API Users. Supported values are: ...` or something like that.